packages/@dcl/inspector/src/components/Renderer/Renderer.tsx

```diff
@@ -49,11 +49,13 @@ const Renderer: React.FC = () => {
         const sceneEntity = sdk.sceneContext.getEntityOrNull(entity)
         if (!sceneEntity) continue
 
-        if (!fileSet.has(value.src)) removeGltf(sceneEntity)
-        else loadGltf(sceneEntity, value.src)
+        removeGltf(sceneEntity)
+        if (fileSet.has(value.src)) {
+          loadGltf(sceneEntity, value.src)
+        }
```

Comment:
I changed this logic to also remove the GLTF before loading it, to match the logic in babylon